content/docs/command-reference/remote/modify.md

```diff
@@ -246,8 +246,8 @@ methods that are performed by DVC (`list_objects_v2` or `list_objects`,
   $ dvc remote modify myremote listobjects true
   ```
 
-- `sse` - server-side encryption algorithm to use (e.g. AES256, aws:kms). By
-  default, no encryption is used.
+- `sse` - server-side encryption algorithm to use (e.g. `AES256`, `aws:kms`,
+  `SSE-C`). By default, no encryption is used.
```

Comment:
Not sure that's the right value. Asked in https://github.com/iterative/dvc.org/pull/3498#pullrequestreview-966849021 ⌛